### ISSUE TITLE
Auto send leftover items to trash

### ISSUE THREAD

**Issue by joshuamking** [Yousei9/Logistic-Train-Network#285]
**TL;DR**
Automatically send trains with leftover items to a specified stop.

**What ?**
So when our base's power gets low or goes out for whatever reason, or we misconfigure a stop, etc.. we end up with random items left on trains which gets rather annoying. It would be nice to be able to automatically send those items to our "Trash" stop which is simply a bunch of inserters and active providers to empty the trains. Right now this is a manual process but it would be cool if rather than (or in addition to) alerting that items were left of the train, it sends it to a stop that is marked as a "Trash" stop (like marking stops as depos works). 

Secondly, our outposts get cluttered with random unnecessary items. So I've "jerry-rigged" a "Send to trash" stop at our outposts to say that when it has items, that it can provide rocket silos and I marked my trash stop as perpetually requesting rocket silos so that any items in the "Send to trash" stop are sent to the "Trash" stop. Lol not pretty, but it works. It'd be really cool to be able to also set up a "Send to trash" concept (maybe that should be another GH issue?). This needs more thought but it would be super nice.

**Why ?**
This will allow new users to LTN a bit of grace when setting up their stops and would allow all users peace of mind that their trains are never going to be sitting with leftover cargo at their depots.

I would love to create a PR with these changes but wanted to bring it up as a topic first.
What are your thoughts? 

@Yousei9

**Comment by Yousei9** [comment]
This came up before, there's even a mod doing that https://mods.factorio.com/mod/ltn-cleanup

My reply is, why do you need yet another stop to do what is intended to be done in the depot!?
1) Providers shouldn't load random trash into trains to begin with.
If blackouts cause your problems don't ask for a logistics mod to fix your bad power infrastructure.
2) The whole reason why LTN makes Depot > Provider > Requester > Depot round trips is so there is a central point to refuel and clean residue.
Place a high priority provider next to the depot to get the cleaned out items back into the network.
